app/models/ability.rb

```diff
@@ -4,8 +4,11 @@ class Ability
   def initialize(user)
      user ||= User.new
 
+     alias_action :create, :read, :update, :destroy, :to => :crud
+
       if user.is_admin?
-        can :manage, :all
+        can :crud, User
```

Comment:
Here is what response I expected from you :) 
https://github.com/ryanb/cancan/wiki/Defining-Abilities
<img width="863" alt="screen shot 2017-03-27 at 19 38 43" src="https://cloud.githubusercontent.com/assets/1469375/24369718/2ce87ebe-1325-11e7-80c8-52cc820f7f42.png">
>Look at line 9.

Is not an argument or explanation of it.